lib/ach_client/version.rb

```diff
@@ -1,4 +1,4 @@
 module AchClient
   # Increment this when changes are published
-  VERSION = '0.4.0'
+  VERSION = '0.5.0'
```

Comment:
You broke the current interface, should bump to 1.0.0 when that happens if we are following SemVer